zerver/lib/import_realm.py

```diff
@@ -418,6 +419,30 @@ def update_model_ids(model: Any, data: TableData, related_table: TableName) -> N
         update_id_map(related_table, old_id_list[item], allocated_id_list[item])
     re_map_foreign_keys(data, table, 'id', related_table=related_table, id_field=True)
 
+def bulk_import_user_message_data(data: TableData) -> None:
+    model = UserMessage
+    table = 'zerver_usermessage'
+    items = data[table]
+
+    offset = 0
+    chunk_size = 10000
+
+    while True:
+        ums = [
+            UserMessageLite(
+                user_profile_id = item['user_profile_id'],
+                message_id = item['message_id'],
+                flags=item['flags'],
+            )
+            for item in items[offset:offset+chunk_size]
+        ]
+        if not ums:
+            break
+        bulk_insert_ums(ums)
+        offset += chunk_size
+
+    logging.info("Successfully imported %s from %s." % (model, table))
```

Comment:
The logging line could be better (basically, the `dump_file_id` feature of `bulk_import_model` was primarily for this; probably an easy `log_import_successful` function extraction).

Reply:
The function I based this off of has a comment claiming we're trying to deprecate `dump_file_id`.  I just added it here, though, as I agree for the two big tables it's potentially useful.